javascript/react/gotchas/dangerously-set-innerhtml.md

```diff
@@ -32,28 +32,60 @@ aspects:
 
 React gives you the ability to insert *raw* HTML.
 
-Take into  consideration that this practice is definitely to be avoided because it makes your code vulnerable to *cross-site scripting* (XSS).
+Take into consideration that this practice is definitely to be avoided because it makes your code vulnerable to *cross-site scripting* (XSS).
```

Comment:
Add a footnote for XSS

Reply:
Added the following footnote:

[1: XSS]
Cross-Site Scripting (or XSS) is a type of attack in which malicious scripts are injected into otherwise trusted websites. These attacks occur when the attacker uses a web app to send the malicious code to a different unsuspecting user. The most widespread flaw that allows this type of attack is represented by apps that make use of user input in their output, without validating or encoding it.

Reply:
Perfect 👍